src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizardHelper.js

```diff
@@ -171,8 +171,8 @@
                                 value: currentField.id
                             }
                         );
-                        //special case so Amount object is always visible
-                        if (currentField.id.includes('Donation_Amount__c')) {
+
+                        if (currentField.locked) {
```

Comment:
I like that you've moved this into a property of the row. The meaning of "locked" won't be readily apparent to me once I've forgotten the context of this change. How about something like "requiredByBGEUI" or "alwaysRequired"?